pmaweb/templates/components/_dlbox.html

```diff
@@ -1,3 +1,4 @@
+<!-- TODO: May be removed after the redesign since it's part of header/navbar -->
```

Comment:
TODO ?

Reply:
I left because I don't know if you want me to remove or let in there. I didn't find any other place that uses this component.